### PR TITLE
Add Nightly Boards [n]87 and [n]2

### PR DESCRIPTION
**Pull request by NightlyBoards**
## Description

Adding the json files for 2 of my keyboards: [n]87 and [n]2. Both are tested working with the via software

## QMK Pull Request 

n87: https://github.com/qmk/qmk_firmware/pull/9885

n2: https://github.com/qmk/qmk_firmware/pull/9886

## Checklist

- [x] The QMK source code follows the guide here: https://caniusevia.com/docs/configuring_qmk
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

**Comment by NightlyBoards** [comment]
Both boards are now merged on the qmk repo

**Comment by Wilba6582** [comment]
Can you add keys with "decal" checked for the blockers? They do not need a switch matrix coordinate but they do need the layout option "2,1" in the bottom right.
image

File: src/nightly_boards/n87.json [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+  "lighting": "qmk_rgblight",
+  "matrix": {
+    "rows": 12,
+    "cols": 9
+  },
+  "layouts": {
+    "labels": [
+      "Split Backspace",
+      "Split Right Shift",
+      ["Bottom Row","6.25U","Centered 6.25U","Tsangan"]
```
Comment (Wilba6582) [inline src/nightly_boards/n87.json:14]: We prefer "7U" instead of "Tsangan".

Comment (NightlyBoards) [reply src/nightly_boards/n87.json:14]: Okay thanks! Will edit it

Comment (NightlyBoards) [reply src/nightly_boards/n87.json]: https://github.com/the-via/keyboards/pull/241/commits/b8fbb9e3bbed58867221cfaca3e93fab28ac0fa6

**Pull request merged**